### PR TITLE
Fix boxing in async func

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir/issues/807

cc @GuillaumeGomez , @sdroege

### CODE REVIEW

**Comment by EPashkin** [comment]
Ready for review

File: Cargo.toml [hunk 1]
```diff
@@ -34,8 +34,8 @@ v3_24 = ["v3_22", "gtk-source-sys/v3_24"]
 default = ["v3_0"]
 purge-lgpl-docs = ["gtk-rs-lgpl-docs", "gtk/purge-lgpl-docs"]
 embed-lgpl-docs = ["gtk-rs-lgpl-docs", "gtk/embed-lgpl-docs"]
-dox = ["gtk/dox", "gtk-source-sys/dox"]
-futures = ["futures-preview", "fragile", "gio/futures"]
```
Comment (GuillaumeGomez) [inline Cargo.toml]: Good catch for this one!

**Comment by GuillaumeGomez** [comment]
👍 

cc @sdroege

**Comment by sdroege** [comment]
:+1: @GuillaumeGomez all green also

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**